### PR TITLE
Startpage & design

### PR DESCRIPTION
**Pull request by Pesidonvera**

### CODE REVIEW

File: shipcss.css [hunk 1]
```diff
@@ -1,9 +1,62 @@
 :root {
-    --board-color: navy;
-    --border-color: lightskyblue;
+    /* --board-color: navy; */
+    --border-color: lightgray;
 }
+body {
+    background-image: url(../ships/photo/image\ 2.png);
```
Comment (narghar) [inline shipcss.css]: Please change path to `url('../photo/background.png');`
And filename respectively. Please don't use spaces in filenames and paths.

File: statki.html [hunk 2]
```diff
@@ -9,11 +9,20 @@
...
      <link rel="stylesheet" href="shipcss.css">
   </head>
   <body>
+        <div id="startpage">
+            <h1>Statki</h1>
+            <h3>Zapraszamy do najlepszej  wersji</br>przegądarkowej gry w statki!</br></br></br>Wybierz sposób gry:</h3>
+            <button id="onePlayer">Jeden Gracz</button>
+            <button id="twoPlayers">Dwóch Graczy</button>
+            <button id="aiPlayer">Zagraj z AI</button>
+            <img id="radarImg" src="../ships/photo/radar-pngrepo-com.png" alt="radar">
```
Comment (narghar) [inline statki.html]: Please change path to `"../photo/radar.png"`
And filename respectively.

Comment (Pesidonvera) [reply statki.html]: solved

File: shipcss.css (hunk 1 above)
Comment (Pesidonvera) [reply shipcss.css]: solved

**Pull request merged**